docs/modules/ROOT/pages/credits.adoc

```diff
@@ -1,9 +1,15 @@
-= Credits
+= Managing Credits
 
-If you have a promo code then you can use it to redeem credit. As long as your credit balance is positive, any usage will be deducted from your credit balance. When you are out of credit, and if you do not have any other payment method defined, then your clusters will be stopped. 
+If you have a promo code then you can use it to redeem credit. As long as your credit balance is positive, any usage will be deducted from your credit balance. When you are out of credit, and if you do not have any other payment method defined, then your clusters will be stopped.
 
-You can see the remaining credit balance under *Account* > *Billing & Payments* page.
+== Prerequisites
 
-You can redeem credit under Account > Credits page, as shown below:  
+To manage credits for a team, you must have either an admin or finance role. See xref:teams-and-users.adoc[].
```

Comment:
There is adoc extension in link text
![Screen Shot 2021-09-02 at 15 26 21](https://user-images.githubusercontent.com/1237982/131843221-c00e9d99-d989-460b-90c5-5c76e6df93be.png)